Fixed various sulfur library entries

alongd opened the pull request:
- **Fixed SH + SH <=> HSSH in GlarborgH2S kinetic library**
The original study (DOI 10.1002/kin.21055) reported a model in which the
high-P rate of the reaction in the title was deliberately reduced to fit
the experiment (See Fig. 4). This PR assigns the original pre-exponential
factor value back.
- **Added kinetics from Marshall1995 to the SOx library**
- **Added thermo of HSO and HOS from Marhsall1995 to the SOxNOx library**
- **Removed the `SO2(T) <=> SO2` transition from the SOx library**
It should be treated better using the Singlet_Val6_to_triplet library
- **Updated `SO + O <=> SO2` in the SOx library**
- **Updated `HOSO <=> SO + OH` in the SOx library**
- **Added the `Sulfur/H2S+S=2SH_1bar` library**
- **Updated H298 and uncertainties in thermo SOxNOx library**
- **Added the SH + O2 PES from Sendt2009 to the SOx kinetic library**

mjohnson541 commented on input/kinetics/libraries/Sulfur/H2S+S=2SH_1bar/reactions.py:
```diff
@@ -0,0 +1,54 @@
+    label = "H2S + S <=> SH + SH",
+    degeneracy = 1,
+    duplicate = True,
+    kinetics = Arrhenius(A=(3.7e+06, 'cm^3/(mol*s)'), n=2.297, Ea=(9010, 'cal/mol'), T0=(1, 'K'),
```
Why do we have two duplicate reactions with the exact same rates from different sources?

alongd commented:
@mjohnson541, thanks for the critical eye!
- I updated the `H2S+S=2SH_1bar` library with the rate of the second DUP reaction from the correct source. Thanks for catching this.
- Yes, I confirm that H298 of `HSOH` should be ~x0.5 of the original value. I added additional sources that support the new value in the comments of this entry.

Although I expect some more correction to the sulfur database as I continue experimenting with it, if everything else look good here this PR can be merged.
